sim/firesim-lib/src/main/scala/bridges/TerminationBridge.scala

```diff
@@ -0,0 +1,136 @@
+
+package firesim.bridges
+
+import chisel3._
+import chisel3.util._
+import freechips.rocketchip.config.Parameters
+import midas.widgets._
+import midas.targetutils._
+import freechips.rocketchip.util.{DecoupledHelper}
+
+/**
+  * Defines a condition under which the simulator should halt. 
+  * @param Message A string printed by the driver to indicate why the simulator is terminating.
+  * @param isError When true, instructs the driver to return a non-zero value when exiting under this condition.
+**/
+case class TerminationCondition(isErr: Boolean, message : String)
+
+//A seq of termination conditions one for each concurrently running program
```

Comment:
Make this `ScalaDoc` as above. See: https://docs.scala-lang.org/style/scaladoc.html

Reply:
Yup. Will keep in mind the scaladoc style.